### ISSUE TITLE
how to make navbar bootstrap into a megamenu

### ISSUE THREAD

**Issue by fadetia** [understrap/understrap#669]
hello, i want is it capable to create megamenu from bootstrap navbar that used in understrap?

## Issue is about..
    - [✓ ] a Question

**Comment by 0dp** [comment]
are you asking if the theme has a built-in mega menu, or if it's possible to create one with bootstrap?

**Comment by fadetia** [comment]
built-in mega menu actually, i was trying to convert bootsrap nav menu into
wordpress using wp bootstrap 4 navwalker it works well,  but i have no idea
how to creating mega menu with it

**Comment by 0dp** [comment]
Ok, so there is no built-in mega menu in UnderStrap. But with some extra css and maybe a few additional classes in the Walker you should be able to achieve the desired mega menu

**Comment by pattonwebz** [comment]
I've tried in the past to make a full featured mega menu with this walker and I have struggled to make it work with certain aspects like images and descriptions in any real scalable way.

It's needs additional CSS, JS and some tweaks to internal functions of the navwalker.

That being said I would love to help create an extension for the walker than can provide everything needed for such a menu. 

If we detail a specific feature set that would be necessary for it to serve most needs and come up with an outline of how it would work in the wordpress menu editor I might be able to start work on a walker extension to make it happen :)
